README.md

```diff
@@ -292,6 +292,7 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 * [**OsmAnd~**](http://osmand.net/) <sup>**[[F-Droid](https://f-droid.org/app/net.osmand.plus)]**</sup>
     * [Contour Lines and Hillshade](http://osmand.net/features?id=contour-lines-plugin) <sup>**[[F-Droid](https://f-droid.org/app/net.osmand.srtmPlugin.paid)]**</sup>
     * [Parking Spot](http://osmand.net/features?id=parking-plugin) <sup>**[[F-Droid](https://f-droid.org/app/net.osmand.parkingPlugin)]**</sup>
+* [**Trail Sense**](https://kylecorry.com/app/maker/subnautica/2020/05/14/trail-sense.html) <sup>**[[F-Droid](https://f-droid.org/en/packages/com.kylecorry.trail_sense/)]**</sup>
```

Comment:
Trail Sense is included under *Misc* already. But I do not have any preference where to put, so if it suits better here, just remove it in the other section.

Reply:
OK, will do 👍

Reply:
Done. Check the newest commit